cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -394,13 +396,124 @@ func deploymentNameForEnv(envName string, clock clock.Clock) string {
 	return name[len(name)-cArmDeploymentNameLengthMax:]
 }
 
+// prevDeploymentResult looks and finds a previous deployment for the current azd project.
+func (p *BicepProvider) prevDeploymentResult(
```

Comment:
Consider renaming to `latestDeploymentResult` to make it clear that it is the most recent one?